src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/ProcessHistoryItem.cs

```diff
@@ -36,6 +36,11 @@ public class ProcessHistoryItem
         /// The end time for process
         /// </summary>
         public DateTime? Ended { get; set; }
+
+        /// <summary>
+        /// An identifier of the entity that performed the action. Organisation number, person identification number or user name.
+        /// </summary>
+        public string PerformedBy { get; set; }
```

Comment:
Using a single field for all identifiers, should this be seperated into three fields? @SandGrainOne @TheTechArch

Reply:
Would this both include ssn and userid?

Reply:
Anyway this is not stored anywhere so (just as instanceevents) so we could map more ids later?

Reply:
I was thinking only SSN, orgNumber, and username (for self identified)